synapse/federation/transport/client.py

```diff
@@ -352,7 +353,9 @@ def get_public_rooms(
         else:
             path = _create_v1_path("/publicRooms")
 
-            args = {"include_all_networks": "true" if include_all_networks else "false"}
+            args = {
+                "include_all_networks": "true" if include_all_networks else "false"
+            }  # type: Dict[str, Any]
```

Comment:
```suggestion
            }  # type: Dict[str, str]
```

?

Reply:
We end up setting the values to tuples and lists below.

Reply:
Ah, yes we do.